### ISSUE TITLE
Rename `get_transformation_list` to `list_transformations`

### ISSUE THREAD

**Issue by csadorf** [materials-marketplace/python-sdk#46]
I propose to rename the `get_transformation_list()` function to `list_transformations()` for conciseness:
https://github.com/materials-marketplace/python-sdk/blob/32e20649969a7f063d7aae016bddc0f9b8346dd5/marketplace/app/v0/transformation.py#L9

**Comment by pablo-de-andres** [comment]
Shouldn't this better go in the standard app API, since we want to rename the capability?
Which might change because of #41, but that is the original source for the name.

I do agree, though. It is also not consistent with other like the semantic mappings:
https://github.com/materials-marketplace/python-sdk/blob/32e20649969a7f063d7aae016bddc0f9b8346dd5/marketplace/app/v0/object_storage.py#L188

**Comment by csadorf** [comment]
I don't agree that the function names in the SDK and the operationIDs (capabilities) must correspond one-to-one. One is a RESTful API, the other one is a Python library so we should use conventions that are applicable in the respective context.

**Comment by pablo-de-andres** [comment]
I meant this because the capabilities also reflect this discrepancy. There is [`getTransformationList`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L1212) on one hand and [`listSemanticMappings`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L1212) (or [`listCollections`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L174), [`listDatasets`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L252)) on the other.

**Comment by csadorf** [comment]
> I meant this because the capabilities also reflect this discrepancy. There is [`getTransformationList`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L1212) on one hand and [`listSemanticMappings`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L1212) (or [`listCollections`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L174), [`listDatasets`](https://github.com/materials-marketplace/standard-app-api/blob/main/openapi.json#L252)) on the other.

Ah yes, that should probably be renamed as well.
